### PR TITLE
Initial rules

### PR DESCRIPTION
**Pull request by JakeSidSmith**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
```
Comment (manosim) [inline .gitignore]: Maybe `node_modules/`? Looks more ...?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "eslint",
+    "linting",
+    "react",
+    "es6",
+    "javascript",
+    "config",
+    "configuration",
+    "oleous"
+  ],
+  "author": "oleous",
```
Comment (manosim) [inline package.json]: `"author": "Oleous <hello@oleous.com>",`

**Comment by manosim** [comment]
Other than the 2 comments looks good to me :ship: :ship: :ship:

File: package.json (hunk 2 above)
Comment (JakeSidSmith) [inline package.json]: Good idea. :)

**Pull request merged**